### ISSUE TITLE
Report Bugs

### ISSUE THREAD

**Issue by Mudit018** [opencodeiiita/Geek-Editor-Frontend#185]
# In this issue, you have to report all the relevant bugs and if they are viable, we will assign them to you.

## Note: 
- No need to claim.
- If we find a certain bug (reported by you) as an issue that we are planning to give later in this project, then it won't be assigned to you
- Final decision rests in the hands of mentors

**Comment by varunKT001** [comment]
there is a bug is the signin page. the signin button does not work

[video](https://user-images.githubusercontent.com/83509023/139521545-53ba8366-d94e-4c4a-aebd-6d5a3f860cbf.mp4)

**Comment by Mudit018** [comment]
@varunKT001 Yup this is a valid issue, so I am making an issue and you can then start working on it

**Comment by shlesh** [comment]
@Mudit018 @Priyal-Saxena @aknsal For some reason the heroku deployment is not letting me login to the app

**Comment by shlesh** [comment]
![image](https://user-images.githubusercontent.com/76761060/139871993-d3d62ce6-2ada-4356-8f52-d66e5c6a63cc.png)
And i would like to point out a bug/feature request that there is a lack of a `404 page` on the website. If a user enters anything other than the defined routes they get a navbar and a footer, a `404 page` would have made the site complete in this case

**Comment by Priyal-Saxena** [comment]
@shlesh valid issue, I'll be making an issue and you can work on it.
